### PR TITLE
Added a bulgarian translation

### PR DESCRIPTION
**Pull request by hristo-d-hristov**

### CODE REVIEW

**Comment by ypid** [comment]
Thanks. I gave it a quick try. For example it renders to "Това произведение е лицензирано под Creative
Commons “CC0 1.0 Универсален” лиценз.". Is this acceptable that a few words are still in Latin alphabet? I am just asking because in #59 this was made translatable as well.

File: doclicense/doclicense-bulgarian.ldf [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+\@namedef{doclicense@lang@lic@CC@by@4.0}{Признание 4.0 Международен}%
+\@namedef{doclicense@lang@lic@CC@by-sa@4.0}{Признание-Споделяне на споделеното 4.0 Международен}%
+\@namedef{doclicense@lang@lic@CC@by-nd@4.0}{Признание-Без производни 4.0 Международен}%
+\@namedef{doclicense@lang@lic@CC@by-nc@4.0}{Признание-Некомерсиално 4.0 Международен}%
+\@namedef{doclicense@lang@lic@CC@by-nc-sa@4.0}{Признание-Некомерсиално-Споделяне на споделеното 4.0 Международен}%
+\@namedef{doclicense@lang@lic@CC@by-nc-nd@4.0}{Признание-Некомерсиално-Без производни 4.0 Международен}%
+
+\AtBeginDocument{\ifthenelse{\equal{\doclicense@type}{CC}}{%
+    \renewcommand*{\doclicenseLongType}{Криейтив Комънс\xspace}}{}}
```
Comment (ypid) [inline doclicense/doclicense-bulgarian.ldf:26]: Using `\AtBeginDocument` is kind of a hack. Try using `\@namedef{doclicense@lang@lic@CC}{...}`.

**Comment by ypid** [comment]
> The translators of the full license in Bulgarian have used the same approach.

Good idea to check that. That increases the confidence in doing it this way. Because ideally this package should not invent stuff, but just make the upstream license easier to use.

> I guess that you don`t know how to read Bulgarian

No, only German and English. The validation I can do on translations is limited. I basically just review the implementation details and hope that other people speaking the language will step in when needed :) Should be fine.

**Comment by hristo-d-hristov** [comment]
Did I do it correctly?
I started learning LaTeX two days ago and this is the first package I contribute to, so I am not really confident in anything I do.
But I think that I understand it ... I believe that \@namedef{doclicense@lang@lic@CC}{...} in the Bulgarian file actually overrides \@namedef{doclicense@lang@lic@CC}{Creative Commons}%, which is line 125 in doclicense.sty. Is that true?

**Comment by ypid** [comment]
Respect than when you only started with LaTeX two days ago :+1: I tested it, looks good to me. Thanks. Merged. I schedule to make a release in about two weeks.

> Is that true?

Yes.

**Pull request merged**